### PR TITLE
[benchmarkRunner] releasing new version of benchmark runner

### PR DESCRIPTION
**Pull request by sameer-in**
## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [X] Title of the PR starts with chart name (e.g. [mychartname])
- [X] Chart Version bumped for the respective charts
- [X] Changelog updated for the charts with version updates.
- [X] Variables are documented in the README.md (or README.tpl in some charts)
- [X] Check GithubAction checks (like lint) to avoid merge-check stoppers

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

**Review by cgeers — APPROVED** [review]

**Review by noahkraemer — CHANGES_REQUESTED** [review]
You also need to bump the version of the `node-analyzer` chart, and then the reference to the `node-analyzer` chart in the `sysdig-deploy` chart

File: charts/sysdig-deploy/CHANGELOG.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ This file documents all notable changes to Sysdig's sysdig-deploy Helm Chart. Th
 
 ### Minor changes
 * Bumped nodeAnalyzer to 1.7.5
-* Bumped benchmark runner to 1.0.18.2
+* Bumped benchmark runner to 1.0.19.0
```
Comment (noahkraemer) [inline charts/sysdig-deploy/CHANGELOG.md]: Please don't modify old changelog entries, you should add a new entry to the top of this file using the new `sysdig-deploy` version

Comment (noahkraemer) [reply charts/sysdig-deploy/CHANGELOG.md]: Same applies to the `node-analyzer` chart

Comment (sameer-in) [reply charts/sysdig-deploy/CHANGELOG.md]: Done. added new entries with bumped version of `node-analyzer` and `sysdig-deploy`

**Review by noahkraemer — APPROVED** [review]
Looks great, thank you!

**Pull request merged**